packages/cypress/src/index.ts

```diff
@@ -35,19 +35,6 @@ const writeArchives = async ({
   pageUrl,
   viewport,
 }: WriteArchivesParams) => {
-  const bufferedArchiveList = Object.entries(resourceArchive).map(([key, value]) => {
-    return [
-      key,
-      {
-        ...value,
-        // we can't use Buffer in the browser (when we collect the responses)
-        // so we go through one by one here and bufferize them
-        // @ts-expect-error will fix when Cypress has its own package
-        body: Buffer.from(value.body, 'utf8'),
-      },
-    ];
-  });
-
```

Comment:
Similar to Playwright, we can now just [pass along](https://github.com/chromaui/chromatic-e2e/blob/main/packages/playwright/src/makeTest.ts#L78) the archive instead of having to massage it here (this removed code was initially added when we were intercepting network requests client-side with Cypress, so we had to bufferize things after they had come to the server). But now we archive on the server, so this isn't needed.